src/App.tsx

```diff
@@ -11,11 +11,21 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => ({
+      count: state.count + 1,
+    }));
   };
 
   add100 = () => {
-    // write code here
+    let i = 0;
+
+    while (i < 100) {
+      this.setState((state) => ({
+        count: state.count + 1,
+      }));
+
+      i += 1;
+    }
```

Comment:
You don't need to use a while loop here. Just use a functional setState as for `addOne` method
```suggestion
      this.setState((state) => ({
        count: state.count + 100,
      }));
```